packages/dev/serializers/src/glTF/2.0/glTFSerializer.ts

```diff
@@ -54,62 +54,52 @@ export interface IExportOptions {
      * @deprecated Please use removeNoopRootNodes instead
      */
     includeCoordinateSystemConversionNodes?: boolean;
+
+    /**
+     * If set to true it will export skin matrix index as Uint16.
+     */
+    userUint16SkinIndex?: boolean;

```

Comment:
I'm not sure why this needs to be an option. If the indices are big enough that warrants Uint16, it should use Uint16, otherwise use Uint8 if it fits?

Also, if we keep this flag, then it should be `use` not `user`